src/python/blazingmq/dev/it/data/data_metrics.py

```diff
@@ -209,701 +237,327 @@ def check(self, value: Any) -> bool:
-                "queue_cfg_bytes": 0,
-                "queue_cfg_msgs": 0,
-                "queue_confirm_msgs": 0,
-                "queue_confirm_msgs_abs": 0,
-                "queue_confirm_time_avg": GreaterThan(0),
-                "queue_confirm_time_max": GreaterThan(0),
-                "queue_consumers_count": 0,
-                "queue_content_bytes": 96,
-                "queue_content_msgs": 32,
-                "queue_gc_msgs": 0,
-                "queue_gc_msgs_abs": 0,
-                "queue_history_abs": 0,
-                "queue_msgs_current": 0,
-                "queue_msgs_utilization_max": 0,
-                "queue_nack_msgs": 0,
-                "queue_nack_msgs_abs": 0,
-                "queue_nack_noquorum_msgs": 0,
-                "queue_nack_noquorum_msgs_abs": 0,
-                "queue_producers_count": 0,
-                "queue_push_bytes": 0,
-                "queue_push_bytes_abs": 0,
-                "queue_push_msgs": 0,
-                "queue_push_msgs_abs": 0,
-                "queue_put_bytes": 0,
-                "queue_put_bytes_abs": 0,
-                "queue_put_msgs": 0,
-                "queue_put_msgs_abs": 0,
-                "queue_queue_time_avg": GreaterThan(0),
-                "queue_queue_time_max": GreaterThan(0),
-                "queue_reject_msgs": 0,
-                "queue_reject_msgs_abs": 0,
-                "queue_role": 0,
-            }
+    },
+)
+
+TEST_QUEUE_STATS_AFTER_CONFIRM_SINGLE_NODE = _deep_merge(
+    TEST_QUEUE_STATS_EMPTY,
+    {
+        "appIds": {
+            "bar": {
+                "values": {
+                    "queue_bytes_current": 30,
+                    "queue_confirm_time_avg": GreaterThan(0),
+                    "queue_confirm_time_max": GreaterThan(0),
+                    "queue_content_bytes": 96,
+                    "queue_content_msgs": 32,
+                    "queue_msgs_current": 10,
+                    "queue_queue_time_avg": GreaterThan(0),
+                    "queue_queue_time_max": GreaterThan(0),
+                }
+            },
+            "baz": {
+                "values": {
+                    "queue_bytes_current": 63,
+                    "queue_confirm_time_avg": GreaterThan(0),
+                    "queue_confirm_time_max": GreaterThan(0),
+                    "queue_content_bytes": 96,
+                    "queue_content_msgs": 32,
+                    "queue_msgs_current": 21,
+                    "queue_queue_time_avg": GreaterThan(0),
+                    "queue_queue_time_max": GreaterThan(0),
+                }
+            },
+            "foo": {
+                "values": {
+                    "queue_bytes_current": 0,
+                    "queue_confirm_time_avg": GreaterThan(0),
+                    "queue_confirm_time_max": GreaterThan(0),
+                    "queue_content_bytes": 96,
+                    "queue_content_msgs": 32,
+                    "queue_msgs_current": 0,
```

Comment:
Some of the values here are explicitly set to 0, because there are other `appIds` where these values are not 0